src/homework5/mypkg/cities.py

```diff
@@ -0,0 +1,28 @@
+"""Cities."""
+
+
+def cities():
+    """Города.
+
+    Входные данные поступают из файла 'filename' из текущей директории. В
+    файле дан список стран и городов каждой страны. В первой строке этого файла
+    содержится количество стран N. Далее идет N строк, каждая строка начинается
+    с названия страны, затем идут названия городов этой страны. В следующей
+    строке записано число M, далее идут M запросов — названия каких-то M
+    городов, перечисленных выше. Программа возвращает страну(ы), где находится
+    запрошенный город в виде строки.
+    """
+    filename = 'test_cities.txt'
+    relation = dict()
+    output = []
+
+    with open(filename) as f:
+        for _N in range(int(f.readline())):
+            country, *cities = f.readline().split()
+            for city in cities:
+                relation[city] = relation.get(city, []) + [country]
```

Comment:
bad idea with lists sum. better idea to find solution with append. if you use standard library you can use defaultdict